src/main/java/de/tum/in/www1/artemis/domain/Exercise.java

```diff
@@ -156,6 +159,19 @@
     @JsonIgnore
     private Set<ExerciseHint> exerciseHints = new HashSet<>();
 
+    @OneToMany(mappedBy = "exercise", cascade = CascadeType.ALL)
+    @LazyCollection(LazyCollectionOption.FALSE)
```

Comment:
why is this set to false?
this would mean that we load tutor scores automatically every time we load an exercise from the db, which would be really bad.
Also LazyCollection should not be used, see https://stackoverflow.com/a/25997098